src/v2/view-builder/views/ov/ChallengeAuthenticatorOktaVerifyView.js

```diff
@@ -0,0 +1,22 @@
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import { 
+  Body as ChallengeAuthenticatorDataOktaVerifyPushOnlyView
+} from './ChallengeAuthenticatorDataOktaVerifyPushOnlyView';
+import { Body as ChallengeAuthenticatorDataOktaVerifyView } from './ChallengeAuthenticatorDataOktaVerifyView';
+
+export default BaseAuthenticatorView.extend({
```

Comment:
this file needs the `BaseFooter` too, right?
```
Footer: BaseFooter,
```

Reply:
Only the `ChallengeAuthenticatorDataOktaVerifyBody` needs to override `Footer: BaseFooter` as it does not need to show the link for 'Verify with something else` 
`ChallengeAuthenticatorDataOktaVerifyPushOnlyBody` view uses the `AuthenticatorFooter` to show 'Verify with something else`